scripts/benchmarking/visualize.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+
+import pandas as pd
+import plotly.graph_objects as go
+from plotly.subplots import make_subplots
+import argparse
+
+
+def main():
```

Comment:
Display average framerate on the plot, commit hash / branch / date, maybe split each scene into separate .csv and plot on its own batch?